docs/reference/licenses.nuget.org.md

```diff
@@ -0,0 +1,69 @@
+# licenses.nuget.org
+
+## Rationale
+
+With introduction of the [license expressions](nuspec.md#license) a requirement emerged to have a reliable service
+that would provide a reference text for individual license identifiers, exception identifiers or license expressions.
+An additional requirement for this service is to have a stable URL schema, that is not susceptible to the link rot,
+so that we can safely use it to provide backwards compatibility for older clients.
+
+Licenses.nuget.org fulfills that role. Gallery uses it to provide the license text reference for packages that
+specify their license using a license expression. `nuget.exe pack` operation sets the [`licenseUrl`](nuspec.md#licenseurl)
+element to point to licenses.nuget.org to provide backwards compatibility with older clients that don't support
+the `license` element.
+
+## Protocol
+
+Licenses.nuget.org is intended to be viewed by people in their browsers, no machine-readable responses are provided.
+HTTPS protocol must be used and requests constructed in a certain way. It only supports `GET` requests. It accepts
```

Comment:
`and requests constructed` -> `and requests are constructed`

Reply:
Just `are` seems wrong. I implied `must be` there, but didn't want to repeat it. Would `requests are expected to be constructed` be ok?

Reply:
Sorry, it seems I misread the sentence. Both of your suggestions look good to me